pdns/iputils.hh

```diff
@@ -59,23 +59,22 @@
 #define le64toh(x) OSSwapLittleToHostInt64(x)
 #endif
 
-// for illumos
-#ifdef BE_64
+#ifdef __sun
```

Comment:
This will also be true on illumos, but in general isn't it considered bad form to check for specific systems instead of checking for features?